components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/DefaultServiceURLBuilder.java

```diff
@@ -58,6 +58,7 @@ public class DefaultServiceURLBuilder implements ServiceURLBuilder {
     protected boolean mandateTenantedPath = false;
     protected Map<String, String> parameters = new HashMap<>();
     protected Map<String, String> fragmentParams = new HashMap<>();
+    protected boolean skipCustomDomain = false;
```

Comment:
`skipDomainBranding` WDYT?

Reply:
Also let's add a comment here explaining why this flag was added and intended purpose.

Reply:
Updated with [36b998d](https://github.com/wso2/carbon-identity-framework/pull/5700/commits/36b998d3cb7980a87c8d3fabe1bfb956dec6124e)